Generated unit test for dataform fails with `Couldn't build proto file into descriptor pool!`

The generated unit tests for google/cloud/dataform/v1alpha2 fail with `google.cloud.dataform.v1alpha2.CompilationResult.CodeCompilationConfig.vars_: map_entry should not be set explicitly. Use map<KeyType, ValueType> instead.`


```
============================================================================== ERRORS ===============================================================================
_______________________________________________ ERROR collecting tests/unit/gapic/dataform_v1alpha2/test_dataform.py ________________________________________________
tests/unit/gapic/dataform_v1alpha2/test_dataform.py:40: in <module>
    from google.cloud.dataform_v1alpha2.services.dataform import DataformAsyncClient
google/cloud/dataform_v1alpha2/__init__.py:17: in <module>
    from .services.dataform import DataformClient
google/cloud/dataform_v1alpha2/services/dataform/__init__.py:16: in <module>
    from .client import DataformClient
google/cloud/dataform_v1alpha2/services/dataform/client.py:37: in <module>
    from google.cloud.dataform_v1alpha2.services.dataform import pagers
google/cloud/dataform_v1alpha2/services/dataform/pagers.py:27: in <module>
    from google.cloud.dataform_v1alpha2.types import dataform
google/cloud/dataform_v1alpha2/types/__init__.py:16: in <module>
    from .dataform import (
google/cloud/dataform_v1alpha2/types/dataform.py:2250: in <module>
    class QueryWorkflowInvocationActionsResponse(proto.Message):
.nox/unit-3-9/lib/python3.9/site-packages/proto/message.py:271: in __new__
    file_info.generate_file_pb(new_class=cls, fallback_salt=full_name)
.nox/unit-3-9/lib/python3.9/site-packages/proto/_file_info.py:104: in generate_file_pb
    pool.Add(self.descriptor)
E   TypeError: Couldn't build proto file into descriptor pool!
E   Invalid proto descriptor for file "google/cloud/dataform_v1alpha2/types/dataform.proto":
E     google.cloud.dataform.v1alpha2.CompilationResult.CodeCompilationConfig.vars_: map_entry should not be set explicitly. Use map<KeyType, ValueType> instead.
```

--- a/gapic/utils/reserved_names.py
+++ b/gapic/utils/reserved_names.py
@@ -24,7 +24,8 @@ RESERVED_NAMES = frozenset(
         # We CANNOT make exceptions for keywords.
         keyword.kwlist,
         # We make SOME exceptions for certain names that collide with builtins.
-        set(dir(builtins)) - {"filter", "map", "id", "input", "property", "vars", "set"},
+        set(dir(builtins)) - {"filter", "map", "id",
+            "input", "property", "vars", "set"},
         # "mapping" and "ignore_unknown_fields" have special uses
         # in the constructor of proto.Message
         {"mapping", "ignore_unknown_fields"},

--- a/gapic/utils/reserved_names.py
+++ b/gapic/utils/reserved_names.py
@@ -24,7 +24,7 @@ RESERVED_NAMES = frozenset(
         # We CANNOT make exceptions for keywords.
         keyword.kwlist,
         # We make SOME exceptions for certain names that collide with builtins.
-        set(dir(builtins)) - {"filter", "map", "id", "input", "property"},
+        set(dir(builtins)) - {"filter", "map", "id", "input", "property", "vars", "set"},
         # "mapping" and "ignore_unknown_fields" have special uses
         # in the constructor of proto.Message
         {"mapping", "ignore_unknown_fields"},

